src/sql/workbench/services/objectExplorer/common/objectExplorerService.ts

```diff
@@ -611,7 +614,15 @@ export class ObjectExplorerService implements IObjectExplorerService {
 			throw new Error('The object explorer server tree view is already registered');
 		}
 		this._serverTreeView = view;
-		this._serverTreeView.onSelectionOrFocusChange(() => this._onSelectionOrFocusChange.fire());
+		this._treeDisposables.push(this._serverTreeView.onSelectionOrFocusChange(() => this._onSelectionOrFocusChange.fire()));
+	}
+
+	public unregisterServerTreeView(): void {
```

Comment:
>unregisterServerTreeView [](start = 8, length = 24)

Doc comments on all public methods please (don't have to add them for existing ones in this file, but doing at least registerServerTreeView and unregisterServerTreeView would be appreciated)